File: limesz_client/projects/pluto/src/lib/api-providers/generated-api/models/resource-description.ts
/* tslint:disable */
/* eslint-disable */
import { ResourceDescriptionOptions } from '../models/resource-description-options';
import { ResourceProp } from '../models/resource-prop';
export interface ResourceDescription {
  descriptionOptions?: ResourceDescriptionOptions;
  name?: string | null;
  props?: Array<ResourceProp> | null;
  type?: string | null;
}

File: limesz_app/pluto_data/Models/ResourceDescription/ResourceDescription.cs
namespace Pluto.Models.ResourceDescription;

public class ResourceDescription
{
    public string Name { get; set; }
    public string Type { get; set; }
    public List<ResourceProp> Props { get; set; }
    public ResourceDescriptionOptions DescriptionOptions { get; set; } = new ResourceDescriptionOptions();
}